src/Compute/Compute/help/Get-AzHost.md

```diff
@@ -0,0 +1,159 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Compute.dll-Help.xml
+Module Name: Az.Compute
+online version: https://docs.microsoft.com/en-us/powershell/module/az.compute/get-azhost
+schema: 2.0.0
+---
+
+# Get-AzHost
+
+## SYNOPSIS
+Get or list hosts.
+
+## SYNTAX
+
+### DefaultParameter (Default)
+```
+Get-AzHost [-ResourceGroupName] <String> [-HostGroupName] <String> [[-Name] <String>] [-InstanceView]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceIdParameter
+```
+Get-AzHost [-ResourceId] <String> [-InstanceView] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+This cmdlet will get a host in a host group.
+This cmdlet also lists all hosts in a host group if a host name is not given.
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> Get-AzHost -ResourceGroupName $resourceGroupName -HostGroupName $hostGroupName -Name $hostName
```

Comment:
@hyonholee would you mind adding the corresponding output for the commands being run in the examples for the new cmdlets? This will let users see what is returned when running the commands.